documentation/developers/2_Branching-Policy.md

```diff
@@ -25,6 +25,8 @@ Branches should be named using the following pattern `category/<reference>/descr
 - `tech-debt` is for technical debt
 - `doc` is for documentation changes
 - `prototype` is for prototype changes
+- `dependabot` is for managing dependencies by Dependabot
+- `chore` is for minor admin tasks, such as toggling features or bumping dependencies
```

Comment:
Minor:

1. is it worth linking to [Dependabot](https://docs.github.com/en/code-security/getting-started/dependabot-quickstart-guide) in the above line?
2. Dependabot itself uses `build(deps)` by default in its brances; is it worth using somethin similar (e.g. `deps`) for consistency?

Reply:
1. Added
2. It seems Dependabot creates branches starting `dependabot/` in this repo (see recently closed relevant PRs) so I think we're OK. This was already in the regex on the branch validation task.